builder/source/simple.go

```diff
@@ -137,7 +138,17 @@ func (s *SimpleSource) download(destination string) error {
 		req.SetChecksum(sha256.New(), sum, false)
 	}
 
-	resp := grab.NewClient().Do(req)
+	// Keep go-grab defaults but also disable compression
+	// https://github.com/cavaliergopher/grab/blob/v3.0.1/v3/client.go#L53
+	tr := &http.Transport{
+		DisableCompression: true,
+		Proxy:              http.ProxyFromEnvironment,
+	}
+
+	client := grab.NewClient()
+	client.HTTPClient = &http.Client{Transport: tr}
```

Comment:
Why not go all the way? Seems the only thing we're keeping by using `NewClient` is the user agent:

```suggestion
	// Create a client with compression disabled.
	// See: https://github.com/cavaliergopher/grab/blob/v3.0.1/v3/client.go#L53
	client := &grab.Client{
		UserAgent: "solbuild",
		HTTPClient: &http.Client{
			Transport: &http.Transport{
				DisableCompression: true,
				Proxy:              http.ProxyFromEnvironment,
			}
		}
	}
```